overrides/config/jei/itemBlacklist.cfg

```diff
@@ -2534,13 +2534,19 @@ advanced {
         appliedenergistics2:facade:{damage:5,item:"undergroundbiomes:metamorphic_stone_emerald_ore"}
         enderio:block_holy_fused_glass:7
         enderio:block_holy_fused_glass:8
+        bibliocraft:fancyworkbench:0
         extracells:pattern.fluid:meat;m=0
         enderio:block_holy_fused_glass:9
+        bibliocraft:fancyworkbench:1
         appliedenergistics2:facade:{damage:2,item:"undergroundbiomes:igneous_brick"}
         appliedenergistics2:facade:{damage:2,item:"undergroundbiomes:igneous_stone_tile.thermalfoundation.ore.mithril.name"}
+        bibliocraft:fancyworkbench:2
         appliedenergistics2:facade:{damage:2,item:"undergroundbiomes:metamorphic_overgrown_snowed"}
         appliedenergistics2:facade:{damage:4,item:"twilightforest:tower_wood"}
         enderio:item_broken_spawner:Greater Dread Spawn
+        bibliocraft:fancyworkbench:3
+        bibliocraft:fancyworkbench:4
```

Comment:
The Dark Oak version(metadata 5) was not hidden in JEI, not sure if that was intentional or not